### ISSUE TITLE
major spellpower weapon enchant should drop from razaani spell thieves prior to 2.1 content.

### ISSUE THREAD

**Issue by quigiboo** [chromiecraft/chromiecraft#4741]
### What client do you play on?

enUS

### Faction

Both

### Content Phase:

61-64

### Current Behaviour

Currently the Bashir spell thieves drop the weapon enchant, which is how it should be after 2.1 (and therefore 3.3.5). 

### Expected Blizzlike Behaviour

If the server is to be progressive similar to other changes made recently then weapon enchant should drop from razaani spell thieves until 2.1 content (black temple). 

### Source

https://wowwiki-archive.fandom.com/wiki/Enchant_Weapon_-_Major_Spellpower

### Steps to reproduce the problem

Kill Bashir spell thieves until they drop the formula.

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/compare/c05c45c51d3d...3ad652d8ab96

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna](https://github.com/azerothcore/mod-eluna)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
- [mod-transmog](https://github.com/azerothcore/mod-transmog)
- [mod-weekend-xp](https://github.com/azerothcore/mod-weekend-xp)
- [mod-instanced-worldbosses](https://github.com/nyeriah/mod-instanced-worldbosses)
- [mod-zone-difficulty](https://github.com/azerothcore/mod-zone-difficulty)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-custom-worldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by BeefSteaker** [comment]
![image](https://user-images.githubusercontent.com/11332559/210370304-8591db22-1e0e-47fd-a2e8-043f4e9e8ef1.png)
https://web.archive.org/web/20070610095931/http://forums.worldofwarcraft.com/thread.html?topicId=94849591&postId=947510958&sid=1

**Comment by The-GhostRider** [comment]
I am closing the Report because the issue is obsolete, but above all it is not valid because as demonstrated by BeefSteaker's comment as well as the attached link, the report is not valid.

**Comment by heyitsbench** [comment]
> I am closing the Report because the issue is obsolete.

Technically the server has not progressed to the point of patch 2.1, so this is not true, nor is it relevant IMO.

> [...] it is not valid because as demonstrated by BeefSteaker's comment as well as the attached link, the report is not valid.

Also not true. Beef was corroborating the 2.1 adjustment to the enchant drop source, further validating the report.

**Comment by The-GhostRider** [comment]
> Technically the server has not progressed to the point of patch 2.1, so this is not true, nor is it relevant IMO.

Yes, you're right the server hasn't reached BT yet, so for that reason you might be right (nitpicking).

The fact that it is "not relevant to you" is also irrelevant.

> Also not true. Beef was corroborating the 2.1 adjustment to the enchant drop source, further validating the report.

It is written that:
**Enchant Weapon- "Major Spellpower" enchant has been stolen from the Razaani spell thieves by the Bash'ir spell thieves.** 

But at this point it says that:
**The Bash'ir are more likely to have the recipe than the Razaani previously did, however.**

This means they can be dropped by both, but are more droppable by Bash'ir than Razaani

mentre il report lamenta che: 
>Current Behaviour
>Currently the Bashir spell thieves drop the weapon enchant, which is how it should be after 2.1 (and therefore 3.3.5).

This doesn't mean Razaans don't drop the item.

**Comment by heyitsbench** [comment]
> This means they can be dropped by both

The term stolen in this context implies that the ability for the item to be dropped has been moved, not duplicated. But we don't have to trust my English interpretation, we can look at a bit more evidence. Taking a look at the [TBC](https://www.wowhead.com/tbc/npc=20614#drops;q=enchant), [Wrath](https://www.wowhead.com/wotlk/npc=20614#drops;q=enchant), or even [retail](https://www.wowhead.com/npc=20614#drops;q=enchant) Wowhead pages for this creature (the Razanni Spell-Thief) shows no dropped enchant.

So, to recap:

Pre-2.1:
- Bash'ir Spell-Thief: No enchant drop.
- Razanni Spell-Thief: Enchant drop, low chance.

Post-2.1:
- Bash'ir Spell-Thief: Enchant drop, medium chance.
- Razanni Spell-Thief: No enchant drop.
